tools/check_environment.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python
```

Comment:
Can this and the other tool be imported from `baselayer`? @stefanv

Reply:
Sure, I just want to keep the deps of `fritz` at the very minimum, but don't really care as it's already using pyjwt and pyyaml (I originally wanted stdlib stuff only)

Reply:
baselayer is already there via skyportal submodule, so yes.

Reply:
In baselayer, it's a stand-alone script that is used in `make dependencies`, and I wanted an importable function. Plus, it would say things like "Please refer to https://cesium-ml.org/baselayer for installation instructions." :)